src/figuresClasses.ts

```diff
@@ -1,11 +1,98 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle';
```

Comment:
The `shape` property should be defined as a constant string literal type, not just a string. It should be `shape: 'triangle' = 'triangle';` to ensure it matches the `Figure` interface.